README.md

```diff
@@ -6,87 +6,66 @@ Deis (pronounced DAY-iss) is an open source PaaS that makes it easy to deploy an
 
 # About
 
-This package provides a the Deis Builder, a git server to respond to `git push`es from clients. When it receives a push, it takes the following high level steps:
+The builder is primarily a git server that responds to `git push`es from clients. When it receives a push, it takes the below high level steps in order.
 
-1. Accepts the code and writes to the local file system
+1. Accepts the code and writes it to the local file system
 2. Calls `git archive` to produce a tarball (i.e. a `.tar.gz` file) on the local file system
-3. Saves the tarball according to the following rules:
-  - If the `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT` environment variables exist, uses the [`mc`](https://github.com/minio/mc) client to save to the [Minio](https://github.com/minio/minio) server at `http://$DEIS_MINIO_SERVICE_HOST:$DEIS_MINIO_SERVICE_HOST`
-  - Otherwise, if the `DEIS_OUTSIDE_STORAGE_HOST` and `DEIS_OUTSIDE_STORAGE_PORT` environment variables exist, uses the [`mc`](https://github.com/minio/mc) client to save to S3 server (or server that adheres to the S3 API) at `https://$DEIS_OUTSIDE_STORAGE_HOST:$DEIS_OUTSIDE_STORAGE_PORT` (this functionality is currently waiting for merge at https://github.com/deis/builder/pull/21).
-4. Starts a builder pod according to these rules:
-  - If a `Dockerfile` is present, starts a [`dockerbuilder`](https://github.com/deis/dockerbuilder) pod, configured to download the code to build from the URL computed in the previous step (`dockerbuilder` and Dockerfile builder are not currently supported. See https://github.com/deis/dockerbuilder/pull/1 for prototype `dockerbuilder` code).
+3. Saves the tarball to centralized object storage according to the following rules:
+  - If the `DEIS_MINIO_SERVICE_HOST` and `DEIS_MINIO_SERVICE_PORT` environment variables exist, saves to the [S3 API](http://docs.aws.amazon.com/AmazonS3/latest/API/APIRest.html) compatible server at `http://$DEIS_MINIO_SERVICE_HOST:$DEIS_MINIO_SERVICE_HOST`
+  - Otherwise, if the `DEIS_OUTSIDE_STORAGE` environment variable exists, saves to `https://$DEIS_OUTSIDE_STORAGE`.
+4. Starts a new [Kubernetes Pod](http://kubernetes.io/docs/user-guide/pods/) to build the code, according to the following rules:
+  - If a `Dockerfile` is present in the codebase, starts a [`dockerbuilder`](https://github.com/deis/dockerbuilder) pod, configured to download the code to build from the URL computed in the previous step.
   - Otherwise, starts a [`slugbuilder`](https://github.com/deis/slugbuilder) pod, configured to download the code to build from the URL computed in the previous step.
 
-# Hacking Builder
+# Development
+
+As mentioned above, we welcome contributions from all developers. The high level process for development matches many other open source projects. See below for an outline.
```

Comment:
Maybe drop "As mentioned above,"? It's not strictly necessary and may be wrong if the "Beta Status" section is rewritten.

Also, I think it's important to avoid first-person pronouns like "I" and "we" in this context. I would prefer:
```
Deis welcomes contributions from all developers.
```
or 
```
The Deis project welcomes contributions from all developers.
```